djangoUImanager/README.md

```diff
@@ -0,0 +1,14 @@
+# BrainKB UI 
```

Comment:
Perhaps a bit of cleanup for this file? It doesn't seem to instruct much

Reply:
now readme just contains the pointer to brainkbdocs